packages/access-api/src/service/access-confirm.js

```diff
@@ -18,7 +19,9 @@ export function parse(invocation) {
   const capability = invocation.capabilities[0]
   // Create a absentee signer for the account that authorized the delegation
   const account = Absentee.from({ id: capability.nb.iss })
-  const agent = Verifier.parse(capability.nb.aud)
```

Comment:
@gobengo

Reply:
It was to appease the fact that I wanted to make `opts.agent` on createSessionProofs of type `DID<'key'>` since that's all I ever expect it to be. I relaxed that to `DID` and so changed this back to what it was too.